### PR TITLE
Add CoC

### PR DESCRIPTION
**Pull request by RichiH**
@brian-brazil @SuperQ @robskillington - Consensus on this, I presume?

Closes: https://github.com/OpenObservability/OpenMetrics/issues/44

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,6 @@ See [our spec file](OpenMetrics.md) and [our proto](proto/openmetrics_data_model
 # Mailing list
 
 Join the [mailing list](https://groups.google.com/forum/m/#!forum/openmetrics) or follow us on [Twitter](https://twitter.com/OpenMetricsIO)
+
+# Code of Conduct
+To make Prometheus a welcoming and harassment-free experience for everyone, we follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```
Comment (SuperQ) [inline README.md:17]:
```suggestion
To make OpenMetrics a welcoming and harassment-free experience for everyone, we follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```

Comment (brian-brazil) [inline README.md:17]: Prometheus?

**Comment by SuperQ** [comment]
Hrm, looks like we might need to disable/re-enable circleci builds.

**Comment by RichiH** [comment]
Heh, yes, guess where I stole it from.

But consensus achieved, correct?

**Comment by SuperQ** [comment]
Yes, I'm happy with it.

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**